src/main/scala/beam/sim/population/PopulationAttributes.scala

```diff
@@ -139,10 +144,37 @@ case class AttributesOfIndividual(
     valueOfTime * generalizedTime
   }
 
+  private def getSituationForVOT(
+    destinationActivity: Option[Activity],
+    originActivity: Option[Activity]
+  ): Set[SituationMultiplier] = {
+    (destinationActivity, originActivity, age) match {
+      case (Some(origin), Some(destination), Some(travelerAge)) =>
```

Comment:
Seems like `origin` and `destination` are swapped. I would also put `originActivity` before `destinationActivity` in each method parameter list.

Reply:
good point